cmd/juju/charmhub/convert_test.go

```diff
@@ -333,7 +342,7 @@ func (filterSuite) TestFilterChannels(c *gc.C) {
 	}}
 	for k, v := range tests {
 		c.Logf("Test %d %s", k, v.Name)
-		_, got, err := filterChannels(v.Input, v.Arch, v.Base)
+		_, got, err := filterChannels(v.Input, v.Arch, noRisk, noRevision, noTrack, v.Base)
```

Comment:
Do we have enough test coverage here? We're using table tests but just passing one a fixed set of values for risk, rev, track etc.

Reply:
Fixed.